template/src/setupTests.js

```diff
@@ -2,4 +2,4 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom/extend-expect';
+import '@testing-library/jest-dom';
```

Comment:
This is done as per the code present in default [cra-template](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/src/setupTests.js)